MAINTAINERS.md

```diff
@@ -17,5 +17,6 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 | Tejas Shah              | [shatejas](https://github.com/shatejas)               | Amazon      |
 | Vamshi Vijay Nakkirtha  | [vamshin](https://github.com/vamshin)                 | Amazon      |
 | Vijayan Balasubramanian | [VijayanB](https://github.com/VijayanB)               | Amazon      |
+|Vikasht34 | [Vikasht34](https://github.com/Vikasht34)| Amazon|
```

Comment:
Can we fix this for consistency?

Reply:
Which inconsistency you are taking about here? Is it the space?

Reply:
Fixed